PhysicsTools/PatAlgos/python/tools/pfTools.py

```diff
@@ -316,10 +316,10 @@ def switchToPFJets(process, input=cms.InputTag('pfNoTauClones'), algo='AK4', pos
     print "input collection: ", input
 
     if algo == 'AK4':
-        genJetCollection = cms.InputTag('ak4GenJetsNoNu'+postfix)
+        genJetCollection = cms.InputTag('ak4GenJets'+postfix)
```

Comment:
This was not changed by Dinko in #7488, but you did it yourself with the ak4 transition #3631.
So the default for PAT was "NoNu" and is now changed to "GenJets". So RECO and PAT used to be inconsistent and are consistent now.
Now I wonder if the PAT users expect to get "NoNu" or "GenJets" as default?